crypto/ake/enclave/src/lib.rs

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) 2018-2020 MobileCoin Inc.
 
 #![no_std]
+#[macro_use]
```

Comment:
This isn't necessary, both because this crate should be `edition = "2018"`, but also because the `vec!` is unnecessary.